doc/classes/RayCast3D.xml

```diff
@@ -136,6 +136,13 @@
 		<member name="collision_mask" type="int" setter="set_collision_mask" getter="get_collision_mask" default="1">
 			The ray's collision mask. Only objects in at least one collision layer enabled in the mask will be detected. See [url=https://docs.godotengine.org/en/latest/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
+		<member name="debug_shape_color" type="Color" setter="set_debug_shape_color" getter="get_debug_shape_color" default="Color( 0.5, 0.7, 1.0 )">
+			Custom color for debug shape. This color will be hightlighted at runtime if raycast is colliding.
+		</member>
+		<member name="debug_shape_thickness" type="int" setter="set_debug_shape_thickness" getter="get_debug_shape_thickness" default="1">
+			Debug shape helps to visualize the raycast during development.
+			When debug_shape_thickness is [code]1[/code], a line is used as debug shape. Otherwise, a parallelepiped will be drawn. The debug shape is only visible in the running project if "Visible Collision Shapes" is checked in the debug menu.
```

Comment:
```suggestion
			If [member debug_shape_thickness] is equal to [code]1[/code], a line is used as th edebug shape. Otherwise, a parallelepiped will be drawn to represent the [RayCast3D]. The debug shape is only visible in the running project if [b]Visible Collision Shapes[/b] is checked in the [b]Debug[/b] menu.
```

Reply:
little typo: **th edebug**

Reply:
@fabriceci Fixed :slightly_smiling_face: